testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/Constants.java

```diff
@@ -113,4 +113,9 @@ public enum AuthScheme {
         BEARER,
         POP
     }
+
+    public static final String STATE = "state";
+    public static final String CODE_LENGTH = "code_length";
+    public static final String SENT_TO = "sent_to";
+    public static final String CHANNEL = "channel";
```

Comment:
Please add some short comments/javadoc describing these and mentioning that these are related to Native Auth (given that these are placed in shared class)